syntax = "proto3";

package helloworld;

service Greeter {
	rpc SayHello(HelloRequest) returns (HelloReply) {}
	rpc SayHellos(HelloRequest) returns (stream HelloReply) {}
	rpc SayHellosClient(stream HelloRequest) returns (HelloReply) {}
}

message HelloRequest { string name = 1; }
message HelloReply { string message = 1; }